extensions/webjars-locator/deployment/src/main/java/io/quarkus/webjar/locator/deployment/WebJarLocatorStandaloneBuildStep.java

```diff
@@ -0,0 +1,58 @@
+package io.quarkus.webjar.locator.deployment;
+
+import java.util.Map;
+
+import org.webjars.WebJarAssetLocator;
+
+import io.quarkus.deployment.Capabilities;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.vertx.http.deployment.RouteBuildItem;
+import io.quarkus.vertx.http.runtime.HttpBuildTimeConfig;
+import io.quarkus.webjar.locator.runtime.WebJarLocatorHandler;
+
+public class WebJarLocatorStandaloneBuildStep {
+
+    @BuildStep()
+    public void staticInit(Capabilities capabilities,
+            HttpBuildTimeConfig httpConfig,
+            BuildProducer<WebJarLocatorConfigBuildItem> standalone) throws Exception {
+        if (capabilities.isCapabilityPresent(Capabilities.SERVLET)) {
```

Comment:
Why is this not needed for Servlet?

Reply:
Can you please elaborate on this? I am sure you have some good technical reason in mind, but I am also sure that Quarkus has a lot up it's sleeve to lift any such restrictions

Reply:
Isn't the reverse needed? if servlet compability is _not_ present, return?

Reply:
I based this extension on the resteasy-extension: https://github.com/quarkusio/quarkus/blob/0d29a66fe75b43dc9914c738e4ceb876272d0d74/extensions/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyStandaloneBuildStep.java#L62-L103. I figured if `RESTeasy` has it for a reason, this extension will probably have it for the same reason. It may be entirely unneeded.